web/package.json

```diff
@@ -11,13 +11,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "@cycjimmy/jsmpeg-player": "^5.0.1",
     "date-fns": "^2.21.3",
     "idb-keyval": "^5.0.2",
     "immer": "^8.0.1",
     "preact": "^10.5.9",
     "preact-async-route": "^2.2.1",
     "preact-router": "^3.2.1",
-    "video.js": "^7.11.8",
+    "video.js": "^7.13.0",
```

Comment:
@hunterjm were you intentionally running an older version? `npm audit fix` updated this

Reply:
Nope. It was probably the latest version when I added it.